packages/typespec-ts/test/integration/routes.spec.ts

```diff
@@ -32,13 +35,100 @@ describe("RoutesClient Rest Client", () => {
     assert.strictEqual(result.status, "204");
   });
 
-  it.skip("should have PathParameters SimpleExpansion Standard primitive", async () => {
+  it("should have allowReserved: true", async () => {
     const result = await client
-      .path("/routes/path/simple/standard/primitive{param}", "a")
+      .path("/routes/path/reserved-expansion/template/{param}", {
+        value: "foo/bar baz",
+        allowReserved: true
+      })
       .get();
     assert.strictEqual(result.status, "204");
   });
 
+  it("should have allowReserved: true with helper", async () => {
+    const result = await client
+      .path(
+        "/routes/path/reserved-expansion/template/{param}",
+        buildAllowReserved("foo/bar baz")
+      )
+      .get();
+    assert.strictEqual(result.status, "204");
+  });
+
+  it("should have explode: true array", async () => {
+    const result = await client
+      .path("/routes/query/query-expansion/explode/array")
+      .get({
+        queryParameters: {
+          param: {
+            value: ["a", "b"],
+            explode: true,
+            style: "form"
+          }
+        }
+      });
+    assert.strictEqual(result.status, "204");
+  });
+
+  it("should have explode: true record", async () => {
+    const result = await client
+      .path("/routes/query/query-expansion/explode/record")
+      .get({
+        queryParameters: {
+          param: {
+            value: { a: 1, b: 2 },
```

Comment:
what would the serialize result would be for record of objects?

Reply:
as above mentioned.